online-banking-cz/src/main/java/com/adyen/checkout/onlinebankingcz/IssuersExt.kt

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
This file is duplicated now, one in issuer list as well, right?

Reply:
no it's for another model as I created a new `OnlineBankingModel` data class that extends `SimpleTextListItem`, but yeah it's the same functionality

Reply:
Hmm in that case these methods can be `internal`, also the ones in issuer list